src/main/flight/mixer.c

```diff
@@ -61,14 +61,16 @@
 
 //#define MIXER_DEBUG
 
+void servoMixTable(void);
```

Comment:
please don't do this. Either declare `servoMixTable` in 'servo.h' and include it, or create 'servo-impl.h'.
Important thing is to include header declaring function in file that defines it - that way compiler will warn if declaration does not match definition.
Mismatched function signature can be very hard to discover/debug